src/cc/api/BPF.cc

```diff
@@ -733,6 +736,18 @@ USDT::USDT(const USDT& usdt)
       name_(usdt.name_),
       probe_func_(usdt.probe_func_) {}
 
+USDT::USDT(USDT&& usdt) noexcept
+    : initialized_(usdt.initialized_),
```

Comment:
Since `usdt.initialized_` is set below, do we need this initialization?

Reply:
This is setting the target's value.Below is to reset the value of the origin (that is being moved away)

Reply:
Oh, that is true. Thanks for reminding me of this!